### DISCUSSION TITLE
'Known problems' in docs ?

### DISCUSSION THREAD

**Discussion by Stephanevg** [dataplat/dbatools#9246] [General]
Salutations yall!

I was working with a colleague, and we bumped into the a warning with this message

"WARNING: SQLPS or SqlServer was previously imported during this session. If you encounter weird issues with dbatools, please restart PowerShell, then import dbatools without loading SQLPS or SqlServer first."

We looked at the docs.dbatools.io page, and search for that error message to get a bit more info / context but we couldn't find anything.
What struck me actually, is that there is no 'Known issues' page on the docs website.
Wouldn't it make sense to create one?

**Comment by wsmelton** [comment]
It is a good bit of effort to maintain a page with known issues. We opt to just depend on GitHub issues for those details. We try to pin the issues as we can identify them of common issues you may experience with the module/command(s). Environmental configurations are hard to test against so there will always be scenarios we cannot account for or are one-off/not common to handle.
